### ISSUE TITLE
Unable to load decryption key: AudioKeyError

### ISSUE THREAD

**Issue by okias** [xou816/spot#528]
**Describe the bug**

**To Reproduce**
1. Try play any song

**Expected behavior**
Song plays

**General information:**
 - Distribution: Debian
 - Installation method Flathub
 - Version [e.g. 0.1.0]: latest
 - Device used: destkop

**Additional context**
```
Unable to load decryption key: AudioKeyError
```
per each song. It worked for like 1 hour before it start printing this, tried to log-out and login, but no luck

**Comment by okias** [comment]
ok, now it started playing after 3 lines of errors, so it could be some rate-limiting?

I'm aware that it's hardly fixable from Spot, but maybe Spot could report it to user in some nice manner?

**Comment by xou816** [comment]
> I'm aware that it's hardly fixable from Spot, but maybe Spot could report it to user in some nice manner?

definitely! do you know what emits that log? (Unable to load decryption key)
